Add DECLARE EXTERNAL declaration to USQL sample

Crawl9992 opened the pull request:
Adding DECLARE EXTERNAL statements to the USQL example will help others avoid the common pitfall listed here: https://stackoverflow.com/questions/49302973/passing-dynamic-parameter-to-usql-script-using-data-factory-v2-usql-activity/49310999#49310999

azurecla commented:
Hi __@shriramnat__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (shnatara). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

shriramnat commented:
sign-off

ErikjeMS commented:
#sign-off

Crawl9992 committed:
Add DECLARE EXTERNAL declaration to USQL sample

Adding DECLARE EXTERNAL statements to the USQL example will help others avoid the common pitfall listed here: https://stackoverflow.com/questions/49302973/passing-dynamic-parameter-to-usql-script-using-data-factory-v2-usql-activity/49310999#49310999

MikeRys reviewed (changes requested):
You may want to add a comment that explains that these two DECLARE statements provide default values that can be overwritten by the ADF parameters and that make the script usable outside of ADF submission.
